pkg/pillar/types/global.go

```diff
@@ -284,6 +284,14 @@ const (
 	// network traces for download requests.
 	// Beware: may contain secrets, such as datastore credentials.
 	NetDumpDownloaderHTTPWithFieldValue GlobalSettingKey = "netdump.downloader.http.with.fieldvalue"
+	// LegacyMACAddress : Enables legacy MAC address generation for applications.
+	// The legacy generation is not "that" random and probability of repating
+	// MAC addresses across nodes is high. Later the generation algorithm was
+	// changed and more randomness was introduced, but some applications may be
+	// already configured with already allocated MAC address, and MAC address
+	// change on EVE node upgrade (switch from old generation logic to new one)
+	// can cause problems with the guest network.
+	LegacyMACAddress GlobalSettingKey = "zedrouter.legacy.mac.address"
```

Comment:
@rouming , please, add some documentation about this new option here: https://github.com/lf-edge/eve/blob/master/docs/CONFIG-PROPERTIES.md

Reply:
Also as part of the comment and docs please mention that this is for local network instances.